src/cartesian/ReferenceLine.tsx

```diff
@@ -150,7 +138,19 @@ export const getEndPoints = (
 };
 
 export function ReferenceLine(props: Props) {
-  const { x: fixedX, y: fixedY, segment, xAxis, yAxis, shape, className, alwaysShow, clipPathId } = props;
+  const { x: fixedX, y: fixedY, segment, xAxisId, yAxisId, shape, className, alwaysShow } = props;
+
+  const [{ xAxisMap, yAxisMap, viewBox, clipPathId }] = useContext(ChartLayoutContext);
```

Comment:
What do you think about creating and using the "useChartLayoutContext"???

Reply:
I don't follow. Could you please rephrase your question?

Reply:
If implemented as `export useChartLayoutContext = () => useContext(ChartLayoutContext);`
it seems like there could be a few small benefits.

Reply:
Sure I can do that. I'm wondering what are the benefits?

Reply:
Hmm... It seems that the import statements will be cleaned up and the implementation will be hidden, so it will be free to modify the internal implementation.

Reply:
Done